warning  '/*eslint*/' has no effect because you have 'noInlineConfig' setting in your config (.eslintrc.js#overrides[3])

```bash
> eslint . --ignore-pattern "!.*"


/home/weiran/repo/eslint-plugin-eslint-plugin/docs/rules/consistent-output.md
  16:1  warning  '/*eslint*/' has no effect because you have 'noInlineConfig' setting in your config (.eslintrc.js#overrides[3])
  37:1  warning  '/*eslint*/' has no effect because you have 'noInlineConfig' setting in your config (.eslintrc.js#overrides[3])
```

it was introduced in https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/commit/15ffadae375f9f835b3f3a18b5aa1bbc07e5efbe. it would be great if we can find a way to avoid the warnings. :)

--- a/package.json
+++ b/package.json
@@ -11,9 +11,10 @@
   "license": "MIT",
   "scripts": {
     "lint": "npm-run-all --continue-on-error --aggregate-output --parallel lint:*",
-    "lint:docs": "eslint --no-inline-config \"**/*.md\" && markdownlint \"**/*.md\"",
+    "lint:docs": "markdownlint \"**/*.md\"",
     "lint:eslint-docs": "npm-run-all \"update:eslint-docs -- --check\"",
     "lint:js": "eslint --cache --ignore-pattern \"**/*.md\" .",
+    "lint:js-docs": "eslint --no-inline-config \"**/*.md\"",
     "lint:package-json": "npmPkgJsonLint .",
     "release": "release-it",
     "test": "nyc --all --check-coverage --include lib mocha tests --recursive",

--- a/package.json
+++ b/package.json
@@ -11,9 +11,9 @@
   "license": "MIT",
   "scripts": {
     "lint": "npm-run-all --continue-on-error --aggregate-output --parallel lint:*",
-    "lint:docs": "markdownlint \"**/*.md\"",
+    "lint:docs": "eslint --no-inline-config \"**/*.md\" && markdownlint \"**/*.md\"",
     "lint:eslint-docs": "npm-run-all \"update:eslint-docs -- --check\"",
-    "lint:js": "eslint --cache .",
+    "lint:js": "eslint --cache --ignore-pattern \"**/*.md\" .",
     "lint:package-json": "npmPkgJsonLint .",
     "release": "release-it",
     "test": "nyc --all --check-coverage --include lib mocha tests --recursive",

